target/linux/lantiq/image/Makefile

```diff
@@ -664,6 +664,18 @@ define Device/avm_fritz7360sl
 endef
 TARGET_DEVICES += avm_fritz7360sl
 
+define Device/avm_fritz7412
+  $(Device/AVM)
+  $(Device/NAND)
+  BOARD_NAME := FRITZ7412
+  DEVICE_DTS := FRITZ7412
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 49152k
+  DEVICE_TITLE := 1&1 DSL-Modem / WLAN-Router - FRITZ7412
```

Comment:
Same here - should be "AVM FRITZ!Box 7412"

Reply:
Thanks, you are right! I changed both to "AVM FRITZ!Box 7412".